README.md

```diff
@@ -308,3 +308,20 @@ The following is equivalent:
 
   See more on [this blog post](https://medium.com/@tomchentw/imagemin-lint-staged-in-place-minify-the-images-before-adding-to-the-git-repo-5acda0b4c57e) for benefits of this approach.
 </details>
+
+## Frequently Asked Questions
+
+### Using with JetBrains IDEs *(WebStorm, PyCharm, IntelliJ IDEA, RubyMine, etc.)*
+
+When using the IDE's GUI to commit changes with the `precommit` hook, you might see inconsistencies in the IDE and command line. This is [known issue](https://youtrack.jetbrains.com/issue/IDEA-135454) at JetBrains so if you want this fixed, please vote for it on YouTrack.
+
+Until the issue is resolved in the IDE, you can use the following config to work around it:
+```js
+{
+  "scripts": {
+    "precommit": "lint-staged",
+    "postcommit": "git update-index --again"
+  }
+}
+```
+*Thanks to [Réda Housni Alaoui](https://youtrack.jetbrains.com/issue/IDEA-135454#comment=27-2710654) for this fix!*
```

Comment:
Do you have any thoughts on using Réda's name here without explicit permission?

I feel a little icky putting someone's full name on something without knowing whether they want it, but didn't want to leave them uncredited either.

Reply:
Let’s remove if you don’t feel like it but I liked the fact you credited him :)

Reply:
What about just crediting the comment itself?

ex: `Thanks to [this comment] for the fix!`